packages/litexa/src/parser/skill.coffee

```diff
@@ -272,11 +272,12 @@ class lib.Skill
     @dataTables[table.name] = table
 
   pushSayMapping: (location, source, target) ->
-    if source of @sayMapping and @sayMapping[source] != target
-      # TODO: support localized mappings
-      console.error "duplicate pronounciation mapping for #{source} as #{target}, previously #{@sayMapping[source]}"
-      #throw new ParserError location, "duplicate pronounciation mapping for #{source} as #{target}, previously #{@sayMapping[source]}"
-    @sayMapping[source] = target
+    for mapping in @sayMapping
+      if (mapping.language is location.language) and (mapping.source is source) and (mapping.target is not target)
+        throw new ParserError location, "duplicate pronunciation mapping for #{source} as #{target}
+          in #{location.language} language, previously #{mapping.target}"
+
+    @sayMapping.push({ language: location.language, source, target })
```

Comment:
Rather than this structure:
```
[
  {
    language: 'default',
    source: 'dog',
    target: 'gerbil'
  },
  {
    language: 'default',
    source: 'cat',
    target: 'fur ball'
  }
]
```
would consider using this structure (removes the redundancy of including the language in every mapping, and speeds up iterating/filtering a language):
```
{
  'default': [
    { source: 'dog', target: 'gerbil' },
    { source: 'cat', target: 'fur ball' }
  ]
}
```
Could optionally also consider using `from/to` instead of `source/target` - think that's a little more legible in context of a mapping.